File: src/main/java/com/baulsupp/oksocial/security/InsecureHostnameVerifier.java
package com.baulsupp.oksocial.security;

import javax.net.ssl.HostnameVerifier;
import javax.net.ssl.SSLSession;

public class InsecureHostnameVerifier implements HostnameVerifier {
  @Override public boolean verify(String s, SSLSession sslSession) {
    return true;
  }
}

File: src/main/java/com/baulsupp/oksocial/security/InsecureHostnameVerifier.kt
package com.baulsupp.oksocial.security

import javax.net.ssl.HostnameVerifier
import javax.net.ssl.SSLSession

class InsecureHostnameVerifier : HostnameVerifier {
    override fun verify(s: String, sslSession: SSLSession): Boolean {
        return true
    }
}
